pipelines/vars/forklift_pulpcore.yml

```diff
@@ -1,9 +1,11 @@
+forklift_name: "pipe-pulp-{{ pipeline_version }}-{{ pipeline_os }}"
```

Comment:
why did you have to define that? it should come from `install_base.yml`

Reply:
The *should* did not pan out

Reply:
Hmm. Weird. (I mean, you stopped including it, but still)